pyproject.toml

```diff
@@ -54,11 +54,11 @@ sphinx-rtd-theme                 = "^1.2.0"
 myst-parser                      = "^1.0.0"
 sphinx-notfound-page             = "^0.8.3"
 sphinx-copybutton                = "^0.5.2"
-furo = "^2023.3.27"
-sphinxext-opengraph = "^0.8.2"
-sphinx-autobuild = "^2021.3.14"
-sphinx-design = "^0.4.1"
-sphinx-multiversion = "^0.2.4"
+furo                             = "^2023.3.27"
+sphinxext-opengraph              = "^0.8.2"
+sphinx-autobuild                 = "^2021.3.14"
+sphinx-design                    = "^0.4.1"
+jinja2                           = "^3.1.2"
```

Comment:
So you tried with `sphinx-multiversion` first, but ended up using `sphinx_polyversion.py` instead? What was the problem?

Reply:
I don't remember exactly. But the new script has the following advantages:

- Concurrent builds
- More configuration (cmd args)
- Builds in isolated poetry venvs
- Generating global files like the toplevel `index.html` dynamically
- Maintained
- Cleaner and in some aspects also simpler code in my Opinion

Reply:
And I will be able to better integrate the script into github actions - at least in my mind.